torch/onnx/_internal/exporter/_registration.py

```diff
@@ -273,6 +277,24 @@ def is_registered(self, target: TorchOp) -> bool:
             True if the given op is registered, otherwise False.
         """
         return bool(self.get_decomps(target))
+    
+    def cleanup_registry(self) -> None:
+        """Cleans up the registry to avoid duplicates based on opset version.
+
+        """
+        cleaned_functions = {}
+        for target_or_name, decomps in self.functions.items():
+            # Filter decompositions to only include those with opset_introduced <= opset_version
+            decomps = [d for d in decomps if d.opset_introduced <= self.opset_version]
+
+            # Sort decompositions by opset_introduced in descending order
```

Comment:
nit: use `max` instead of `sort` ... it is more efficient. It is a single linear pass, unlike sort.